solidity/test/LiquidityProtectionSettings.js

```diff
@@ -118,15 +118,31 @@ describe('LiquidityProtectionSettings', () => {
 
         it('should succeed when an admin attempts to add a whitelisted pool', async () => {
             expect(await settings.isPoolWhitelisted.call(poolToken.address)).to.be.false();
+            expect(await settings.poolWhitelist.call()).to.be.ofSize(0);
+
             await settings.addPoolToWhitelist(poolToken.address, { from: admin });
+
             expect(await settings.isPoolWhitelisted.call(poolToken.address)).to.be.true();
+            expect(await settings.poolWhitelist.call()).to.be.equalTo([poolToken.address]);
```

Comment:
Shouldn't this be `deepEqualTo` or something similar?

Reply:
`equalTo` is already a special/new method from `chai-arrays`.